ci.sh

```diff
@@ -97,9 +97,13 @@ test_dub_package() {
     if [ "${DMD:-dmd}" == "gdmd" ] ; then
         echo "Skipping DUB examples on GDC."
     else
+        local abs_build_path="$PWD/$build_path"
         pushd test/dub_package
         for file in *.d ; do
+            # build with host compiler
             dub --single "$file"
+            # build with built compiler (~master)
+            dub --single --compiler="${abs_build_path}" "$file"
```

Comment:
You could either add sub-configurations to dub.sdl or build with DFLAGS to check that -de succeeds.

Reply:
Ok.